app/main.ts

```diff
@@ -20,6 +23,7 @@ function createWindow(): BrowserWindow {
     width: size.width,
     height: size.height,
     webPreferences: {
+      webSecurity: false,
```

Comment:
Why do we need this?

Reply:
because application not allowed to get model files from download directory